packages/react-components/src/components/ImageGallery.tsx

```diff
@@ -157,35 +156,23 @@ export const ImageGallery = (props: ImageGalleryProps): JSX.Element => {
   const renderBodyWithLightDismiss = (): JSX.Element => {
     return (
       <Stack className={mergeStyles(bodyContainer)} onClick={() => props.onDismiss()}>
-        <FocusTrapZone
```

Comment:
It looks like I don't the issue with additional padding anymore. But the background is opaque now. Is it expected?
![image](https://github.com/Azure/communication-ui-library/assets/98852890/1095b6c4-bc3f-41a0-a316-b9487499e9b9)

Reply:
> It looks like I don't the issue with additional padding anymore. But the background is opaque now. Is it expected? ![image](https://user-images.githubusercontent.com/98852890/269340913-1095b6c4-bc3f-41a0-a316-b9487499e9b9.png)

Yes, the background of the ImageGallery modal should be opaque:
![image](https://github.com/Azure/communication-ui-library/assets/107075081/2110f34d-fd03-47b2-8ce3-ec9c9c56b530)